src/Widgets/SeekBar.vala

```diff
@@ -61,35 +61,35 @@ public class Music.SeekBar : Gtk.Grid {
         };
 
         scale = new Gtk.Scale.with_range (Gtk.Orientation.HORIZONTAL, 0, 1, 0.1) {
+            css_classes = {"accent"},
             draw_value = false,
             hexpand = true
         };
-        scale.get_style_context ().add_class (Granite.STYLE_CLASS_ACCENT);
 
-        scale.button_press_event.connect (() => {
-            scale.value_changed.connect (scale_value_changed);
+        // scale.button_press_event.connect (() => {
```

Comment:
Interesting, so you basically check when the scale is pressed... I'm not sure there's a good way to replace that, maybe scale has something built-in. But I'd definitely not use any events nor GtkGesture here.